modules/ROOT/pages/6.0-release-notes-apis.adoc

```diff
@@ -0,0 +1,209 @@
+= TinyMCE 6.0 API changes
+:navtitle: TinyMCE 6.0 API changes
+:description: TinyMCE 6.0 API changes
+:keywords: releasenotes, apis
+
+This section documents the improvements, changes and fixes made to {productname} 6.0 APIs.
+
+It also documents the {productname} 6.0 APIs that have been removed.
+
+
+* xref:apis[APIs]
+* xref:new-and-improved-apis[New and improved APIs]
+* xref:changed-apis[Changed APIs]
+* xref:fixed-apis[Fixed APIs]
+* xref:removed-or-deprecated-apis[Removed or deprecated APIs]
+
+// tag::apis[]
+[[apis]]
+== APIs
+
+[[new-and-improved-apis]]
+=== New and improved APIs
+
+[[editor-annotator-removeall]]
+==== `editor.annotator.removeAll`
+
+The new `editor.annotator.removeAll` API replaces the `removeAll` function built-in to the Comments plugin.
+
+[[editor-options]]
+==== `editor.options` and `tinymce.activeEditor.options`
+
+The new `editor.options` API replaces two old APIs: `editor.settings` and `editor.getParam`.
+
+Along with this new API, the new `tinymce.activeEditor.options` registry replaces `tinymce.activeeditor.settings`.
```

Comment:
```suggestion
Along with this new API, the new `tinymce.activeEditor.options` replaces `tinymce.activeEditor.settings`.
```

Reply:
Removing `registry` here makes this read poorly. It probably should say `API` instead of `registry` though.

Reply:
```suggestion
```
Close enough to a duplicate of the line above, looking at technical accuracy (`editor` = `tinymce.activeEditor`)